packages/flutter/lib/src/services/semantics_service.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' show TextDirection;
+
+import 'package:flutter/rendering.dart' show AnnounceSemanticsEvent;
+
+import 'system_channels.dart';
+
+/// Allows access to the platform's accessibility services.
+///
+/// Events sent by this service are handled by the platform-specific
+/// accessibility bridge in Flutter's engine.
+class SemanticsService {
+  SemanticsService._();
+
+  /// Sends a semantic announcement.
+  ///
+  /// This should be used for announcement that are not seamlessly announced by
+  /// the system as a result of a UI state change.
+  ///
+  /// For example a camera application can use this method to make accessibility
+  /// announcements regarding objects in the viewfinder.
+  static Future<Null> announce(String message, TextDirection textDirection) async {
+    final AnnounceSemanticsEvent event = new AnnounceSemanticsEvent(message, textDirection);
+    await SystemChannels.accessibility.send(event.toMap());
+  }
+}
```

Comment:
this can be in the semantics layer now

Reply:
This was the reason we created the semantics layer